src/BrazeKit-dev.js

```diff
@@ -360,16 +407,16 @@ var constructor = function () {
 
     function logNonPurchaseCommerceEventWithProducts(mpEvent) {
         const commerceEventAttrs = {};
-
+        let eventName = getCommerceEventName(mpEvent.EventCategory);
```

Comment:
I don't see this being reassigned anywhere.

```suggestion
        const eventName = getCommerceEventName(mpEvent.EventCategory);
```